templates/imgproxy/index.ts

```diff
@@ -0,0 +1,71 @@
+import { createTemplate, Services, randomString } from "~templates-utils";
+
+export default createTemplate({
+  name: "imgproxy",
```

Comment:
The `name` is what is displayed as the label in the panel and on the website. Should this be `Imgproxy`? Maybe this project is an exception and should be lower case.

Reply:
Let's leave it lowercase for this, but keep in mind for others. In the "whiteboard" template I made it `Whiteboard`.

Reply:
Sounds good.